nservicebus/upgrades/5to6/index.md

```diff
@@ -10,20 +10,189 @@ related:
 ---
 
 
-## Move to .NET 4.5.2
+## Upgrading to NServiceBus 6
+
+Every solution is different and will encounter unique challenges when upgrading a major dependency like NServiceBus. It's important to plan out an upgrade project and proceed in well defined steps, stopping to ensure that everything is working after each step. Here are a few things to consider when planning an upgrade project.
```

Comment:
"project" is overloaded. ie is it the csproj? considering we talk about "solution" above with i assume is the VS solution?